apps/settings/js/panels/call/call_settings_task_scheduler.js

```diff
@@ -0,0 +1,35 @@
+define(function(require) {
+  'use strict';
+
+  var CallSettingsTaskSchedulerConstant =
+    require('panels/call/call_settings_task_scheduler_constant');
+  var Module = require('modules/base/module');
+  var TaskScheduler = require('modules/base/task_scheduler');
+
+  var CallSettingsTaskScheduler = Module.create(function() {
+    this._taskScheduler = TaskScheduler();
+  });
```

Comment:
Extending from `TaskScheduler` directly would save the effort of defining existing methods like `observe`.